packages/server-boilerplate-middleware/src/xray.ts

```diff
@@ -0,0 +1,11 @@
+import AWSXRay from 'aws-xray-sdk';
+
+export interface IXrayMiddlewareOptions {
+  config?: any[];
```

Comment:
nit. might want to use a more definitive type `AWSXRay.plugins.Plugin[]`

Reply:
good catch, updated.